adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -108,6 +111,7 @@ private AuthenticationConstants() { }
          * Waiting for broker package to be installed, and resume request in broker. 
          */
         public static final int BROKER_REQUEST_RESUME = 2006;
+        public static final int BROWSER_CODE_DEVICE_REGISTER = 2006;
```

Comment:
you should not be using the same number twice, see where this number is coming from

Reply:
File one issue for both.